contrib/init/upstart/docker.conf

```diff
@@ -37,7 +37,7 @@ script
 	if [ -f /etc/default/$UPSTART_JOB ]; then
 		. /etc/default/$UPSTART_JOB
 	fi
-	exec "$DOCKER" -d $DOCKER_OPTS
+	exec unshare -m -- "$DOCKER" -d $DOCKER_OPTS
```

Comment:
As discussed on IRC, no difference, just a short and long version of the
same flag,give the space on the line.
On Apr 23, 2015 12:44 PM, "Tianon Gravi" <notifications@github.com> wrote:

> In contrib/init/upstart/docker.conf
> <https://github.com/docker/docker/pull/12699#discussion_r28980225>:
>
> > @@ -37,7 +37,7 @@ script
> >  	if [ -f /etc/default/$UPSTART_JOB ]; then
> >  		. /etc/default/$UPSTART_JOB
> >  	fi
> > -	exec "$DOCKER" -d $DOCKER_OPTS
> > +	exec unshare -m -- "$DOCKER" -d $DOCKER_OPTS
>
> Why the fully explicit --mount on the others, but only -m on this one?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/12699/files#r28980225>.
>